java.source.base/src/org/netbeans/api/java/source/TreeUtilities.java

```diff
@@ -1928,20 +1928,32 @@ public boolean isPartOfCompoundVariableDeclaration(@NonNull Tree tree) {
     }
 
     /**
-     * Check the tree has compile error in given collection of errors.
+     * Check the tree has compile error in given errors.
      *
      * @param tree : compilation tree
-     * @param errors : Collection of error code
-     * @return true if tree has compile error present in collection of errors.
+     * @param errors : Array of error code
+     * @return true if tree has compile error present in list of errors.
      * @since 2.37.0
```

Comment:
generally only major.minor version should be used if merging to master. "0" at the end in the `spec.version.base` serves as a placeholder for potential release branches.

Reply:
Changes added